setup.py

```diff
@@ -39,13 +38,10 @@ def run_tests(self):
         "scripts/moti_infill_download.py",
         "scripts/moti_insert_files.py",
         "scripts/moti_infill_insert.py",
-    ]
-    + [
-        f"crmprtd/execution/{name}"
-        for name in pkg_resources.resource_listdir("crmprtd", "execution")
     ],
     entry_points={
         "console_scripts": [
+            "crmprtd_dcp=crmprtd.download_cache_process:main",
```

Comment:
I don't *love* the name `crmprtd_dcp`... is there something that's easier to remember that we can name it?

Reply:
Hmm. Tougher than it sounds, without resorting to `download_cache_process`

`crmp_dl_c_proc`? Not exactly easier to remember, and clunkier.

`crmp_pipeline`? "Pipeline" is already taken, to mean various parts of the processing pipeline. But this is quite literally a pipeline too.

`crmp_everything`? Kind of like a "bagel with everything"? (Plus, bonus wink at currently insanely popular movie.)

`crmp_all`? Ditto, except no movie ref.

Reply:
I'm gonna go with `crmprtd_pipeline`. It's easy enough to change later if we don't like it.